public/index.html

```diff
@@ -20,26 +20,9 @@
       name="twitter:description"
       content="A Python Editor for the BBC micro:bit, built by the Micro:bit Educational Foundation and the global Python Community."
     />
-    <% if (process.env.REACT_APP_GA_MEASUREMENT_ID &&
-    (process.env.REACT_APP_STAGE === 'PRODUCTION' || process.env.REACT_APP_STAGE
-    === "STAGING")) { %>
-    <script
-      async
-      src="https://www.googletagmanager.com/gtag/js?id=%REACT_APP_GA_MEASUREMENT_ID%"
-    ></script>
-    <script>
-      window.dataLayer = window.dataLayer || [];
-      window.gaMeasurementId = "%REACT_APP_GA_MEASUREMENT_ID%";
-      window.gtag = function () {
-        window.dataLayer.push(arguments);
-      };
-      gtag("js", new Date());
-      var options = {
-        anonymize_ip: true,
-        cookie_prefix: "%REACT_APP_GA_COOKIE_PREFIX%",
-      };
-      gtag("config", window.gaMeasurementId, options);
-    </script>
+    <% if (process.env.REACT_APP_STAGE === 'PRODUCTION' || process.env.REACT_APP_STAGE === "STAGING") { %>
```

Comment:
I think we should change this condition to a new environment variable that's only set for microbit-foundation GitHub actions builds (so the same condition that installs the theme package). Then internally we can disable analytics for non-prod/staging builds.